lib/studies/reply/model/email_model.dart

```diff
@@ -17,3 +19,46 @@ class Email {
   final String recipients;
   final bool containsPictures;
 }
+
+class InboxEmail extends Email {
+  InboxEmail({
+    int id,
+    String sender,
+    String time,
+    String subject,
+    String message,
+    String avatar,
+    String recipients,
+    bool containsPictures,
+    this.inboxType = InboxType.normal,
+  }) : super(
+          id: id,
+          sender: sender,
+          time: time,
+          subject: subject,
+          message: message,
+          avatar: avatar,
+          recipients: recipients,
+          containsPictures: containsPictures,
+        );
+
+  InboxType inboxType;
+}
+
+// A set of the different mailbox pages that the Reply app
+// contains. An enum would be preferable, but only primitives
+// values are state restorable.
```

Comment:
I agree, an enum would be better: Can you use an enum and store the index of the enum value (available via the `.index` getter on an enum value) in the restoration state? To restore the original enum, you can use that index as an index into the `values` constant of the enum.